index.bs

```diff
@@ -2536,6 +2539,45 @@ To <dfn>get the navigation info</dfn>, given |context| and |navigation status|:
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.close Command ####  {#command-browsingContext-close}
+
+The <dfn export for=commands>browsingContext.close</dfn> command closes a
+[=top-level browsing context=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextCloseCommand = {
+        method: "browsingContext.close",
+        params: BrowsingContextCloseParameters
+      }
+
+      BrowsingContextCloseParameters = {
+        context: BrowsingContext
+      }
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.close">
+The [=remote end steps=] with |command parameters| are:
+
+  1. Let |context id| be the value of the <code>context</code> field of
+     |command parameters|.
+
+  1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+     with |context id|.
+
+  1. Assert: |context| is not null.
```

Comment:
I can still see an assert here. I assume you missed that one to remove?

Reply:
No I kept it intentionally. 

`Assert: |context| is not null.` is also used in other commands in the same file, eg `browsingContext.navigate`, so I assumed the comment was only about the second assert. I can remove it if it is unneeded, but will wait for confirmation.

Reply:
Maybe I also don't understand `Assert` then, but if we cannot get a browsing context from the `id` it's still an `invalid argument`, and should not trigger a bug in the spec.

Reply:
The use of "trying" and the fact that https://w3c.github.io/webdriver-bidi/#get-a-browsing-context returns `error` in the case that the context doesn't exist, means that we really can't get to this point without having a valid context. So the assert is fine.